ui/packages/shared/profile/src/components/DiffLegend.tsx

```diff
@@ -0,0 +1,109 @@
+import {Fragment, useState} from 'react';
+import {Popover, Transition} from '@headlessui/react';
+import {useAppSelector, selectDarkMode} from '@parca/store';
+import {usePopper} from 'react-popper';
+
+const transparencyValues = [-100, -80, -60, -40, -20, 0, 20, 40, 60, 80, 100];
+
+const DiffLegendBar = ({
+  onMouseEnter,
+  onMouseLeave,
+}: {
+  onMouseEnter: () => void;
+  onMouseLeave: () => void;
+}) => {
+  const isDarkMode = useAppSelector(selectDarkMode);
+
+  const newSpanColor = isDarkMode ? '#B3BAE1' : '#929FEB';
+  const getIncreasedSpanColor = (transparency: number) => {
+    return isDarkMode
+      ? `rgba(255, 177, 204, ${transparency})`
```

Comment:
Is it possible to share these color contants/logic from the `shared/functions` package file above?